staubli_rx160_support/urdf/rx160_macro.xacro

```diff
@@ -2,6 +2,14 @@
 <robot xmlns:xacro="http://ros.org/wiki/xacro">
   <xacro:include filename="$(find staubli_resources)/urdf/common_materials.xacro"/>
 
+  <property name="base_mass"   value="50.0" />
+  <property name="link_1_mass" value="30.0" />
+  <property name="link_2_mass" value="20.0" />
+  <property name="link_3_mass" value="20.0" />
+  <property name="link_4_mass" value="10.0" />
+  <property name="link_5_mass" value="1.0" />
+  <property name="link_6_mass" value="0.05" />
```

Comment:
I'm a big fan of using constants instead of magic numbers, but 1) these seem to be only used in the `inertial` tags, and then only once, and 2) their names are too generic.

Either remove them and use the values directly, or prefix them with something like `staubli_rx160_` (as they are a property of that particular robot).